BLaurenB/techtime
# User story 24

#### What's this PR do?
This adds the functionality to view the Admin Freelancers Index page. The admin can view all current and retired freelancers and there is a link to edit the freelancers from this view. 

#### Where should the reviewer start- what are the pertinent changes and the sequence?


Then view the routes.rb file, the Admin freelancers_controller and the admin freelancers_index view page. 

#### How should this be manually tested?
run test rspec ./spec/features/freelancers/admin_can_view_all_freelancers_spec
spin up local server to view as well. Note that the Edit button doesn't not work yet- this is user story 25.

#### Any background context you want to provide?
This is not styled yet.

#### What are the relevant tickets?
closes #125 
#### Screenshots (if appropriate)
#### Questions:
